src/System.ComponentModel.TypeConverter/src/System/Drawing/FontConverter.cs

```diff
@@ -0,0 +1,399 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections;
+using System.ComponentModel;
+using System.ComponentModel.Design.Serialization;
+using System.Drawing.Text;
+using System.Globalization;
+using System.Reflection;
+using System.Text;
+
+namespace System.Drawing
+{
+    public class FontConverter : TypeConverter
+    {
+        public override bool CanConvertFrom(ITypeDescriptorContext context, Type sourceType)
+        {
+            return sourceType == typeof(string) ? true : base.CanConvertFrom(context, sourceType);
+        }
+
+        public override bool CanConvertTo(ITypeDescriptorContext context, Type destinationType)
+        {
+            if (destinationType == typeof(string))
+            {
+                return true;
+            }
+            if (destinationType == typeof(InstanceDescriptor))
+            {
+                return true;
+            }
+            return base.CanConvertTo(context, destinationType);
+        }
+
+        public override object ConvertTo(ITypeDescriptorContext context, CultureInfo culture, object value, Type destinationType)
+        {
+            if ((destinationType == typeof(string)) && (value is Font))
+            {
+                Font font = (Font)value;
```

Comment:
can you move this font declarations inlined to the is statement?

i.e: `value is Font font`